recipes/amos/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: amos
+  version: "3.1.0"
+
+about:
+  home: 'http://amos.sourceforge.net/wiki/index.php/AMOS'
+  summary: 'A Modular, Open-Source whole genome assembler'
+  license: Artistic License
+
+source:
+  git_url: http://git.code.sf.net/p/amos/code
+  git_rev: 9bd658
+
+build:
+  number: 3
+  # OSX failing due to missing dependencies in OSX branch
+  skip: True # [osx]
+
+
+test:
+  commands:
+    - Minimo -h > /dev/null
+
+requirements:
+  build:
+    - gcc
+    - zlib
+    - boost 
+    - autoconf
+    - blat
+    - mummer
+    - perl-statistics-descriptive
+    - perl-xml-parser
```

Comment:
I guess you also need to depend on `perl-threaded`.
@bioconda/core do you think this is necessary or is the implicit dependency on perl enough? I'm more towards explicit.